src/lib/ledger-context.js

```diff
@@ -35,12 +35,17 @@ function parseAndValidateLedgerUrls (metadataUrls) {
 }
 
 class LedgerContext {
-  constructor (host, ledgerMetadata) {
+  constructor (host, ledgerMetadata, overrides) {
+    const metadata = Object.assign({}, ledgerMetadata, overrides || {})
```

Comment:
Maybe for safety, this should throw an error if overrides tries to overwrite a field that already exists. Then again, it _is_ called overrrides.